src/main/haskell/language-kore/src/Data/Kore/Unparser/Unparse.hs

```diff
@@ -108,18 +111,36 @@ instance Unparse (Variable level) where
     unparse var =
         unparse (variableName var) >> write ":" >> unparse (variableSort var)
 
-instance Unparse UnifiedSortVariable where
-    unparse (ObjectSortVariable sv) = unparse sv
-    unparse (MetaSortVariable sv)   = unparse sv
+instance
+    ( Unparse (sort Object)
+    , Unparse (sort Meta)
+    ) => Unparse (Unified sort)
+  where
+    unparse (UnifiedObject s) = unparse s
```

Comment:
doesn't applyUnified or something similar work here? If not, how about an 'applyUnifiedM'?